README.md

```diff
@@ -0,0 +1,204 @@
+# xsane-flatpak
```

Comment:
No user is going to find this README to read your FAQ. If you consider any of this critical, put it in appdata.